### PR TITLE
Introduce TopPHP API wrapper.

### PR DESCRIPTION
**Pull request by goverfl0w**
This would introduce the TopPHP API wrapper library officially to the Top.GG docs page and give an example, as well as calling the new source as "php-sdk." I, unfortunately, have no ability to create the directory myself but approve of whatever motion that carries with it being added in.

### CODE REVIEW

File: content/libraries/php.mdx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+description: Official Top.gg PHP library
+---
+
+# PHP Library
+
+This is our official PHP Library for top.gg, if you have any issues/bugs please submit an issue on our github.
+
+- [GitHub Link](https://github.com/vapourlabs/TopPHP)
+
+(I cannot make a repo for the actual /top-gg/php-sdk version, so a clone will be req.)
```
Comment (Xetera) [inline content/libraries/php.mdx:12]: This doesn't need to be in the docs

**Review by Xetera — COMMENTED** [review]
I'm not a php user but are you not uploading your library to something like packagist? Asking people to clone a repo from github to use dependencies gives me C vietnam flashbacks

File: content/libraries/php.mdx (hunk 1 above)
Comment (goverfl0w) [reply content/libraries/php.mdx:12]: I can make a packagist upload. Will update link ASAP to reflect.

**Comment by Xetera** [comment]
Is this ready to go? Also can you add me or veld as a maintainer to the package on packagist?

**Comment by goverfl0w** [comment]
Yeah, I can get both you and Veld added as maintainers of the package on packagist during this week, I've been busy with school, I apologize for the belated response.

**Comment by braindigitalis** [comment]
can I please suggest some enhancement as a PHP developer myself:

1) support PSR for logger classes instead of just overriding a log method for logging please.
2) why no support for PHP 7? it's nowhere near depreciated yet.
3) personal pet peeve why are some keywords upper case and some not (`REQUIRE`)?

Apart from that looking great!

**Comment by goverfl0w** [comment]
> 1. support PSR for logger classes instead of just overriding a log method for logging please.

Coming this weekend, been tied up with my schedule.

> 2. why no support for PHP 7? it's nowhere near depreciated yet.

Code has been made now for support of PHP 7.4.

> 3. personal pet peeve why are some keywords upper case and some not (`REQUIRE`)?

Code inconsistency has occurred due to bad school practices, will fix formatting problems ASAP.

> Apart from that looking great!

Currently considering adding Symfony as a dependency for HTTP requesting, since doing `stream_create_context()` has had quite a few issues for me. It can either work or break on a webserver depending on the browser you run it on, (guess there's some browser problem?) and it will be easier to understand with an established framework for webserv. `file_get_contents()` is infeasible as of now due to headers being required to pass for certain requests, as well as Webhooks pending integration to this API wrapper.

**Comment by Xetera** [comment]
I don't get it, why do you need to add a webserver as a dependency to send http requests? Also Symfony is not the defacto standard php webserver the way express is so I don't think that's a good idea either way

**Comment by goverfl0w** [comment]
The wrapper is made to be able to work for both webservers (if you want to have a website page show information about bots) or if you want to use it in a bot yourself, to post information to the Top.gg API. I'll really have to look at it because I haven't been able to get to webhooks just yet due to stream contextualization problems, the GET/POST requests work fine with a regular `file_get_contents()` statement without authorization needed.

**Comment by goverfl0w** [comment]
I accidentally had deleted my old fork of this PR in the past, I will close this PR and open a new one with all of the new documentation for the API wrapper re-write.

**Pull request closed without merging**